conf/pillar/testing/env.sls

```diff
@@ -3,8 +3,8 @@ environment: testing
 domain: serviceinfo-testing.caktusgroup.com
 
 repo:
-  url: https://github.com/theirc/ServiceInfo-project.git
-  branch: open_source_project
+  url: https://github.com/theirc/ServiceInfo.git
+  branch: develop
```

Comment:
Is the `testing` environment still supported? I remember seeing the `settings/testing.py` file get deleted in the project repo.